### PR TITLE
Update list.md with official name of SUB Göttingen

### PR DESCRIPTION
**Pull request by nilswindisch**
:octocat:

### CODE REVIEW

File: list.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ librarians.
 Library institutions, journals, and projects
 ---------------------------------------------
 
-SUB Göttingen
+Niedersächsische Staats- und Universität Biliothek Göttingen (SUB Göttingen, SUBUGOE)
```
Comment (stweil) [inline list.md]: _Biliothek_ is not the official name, is it?

Comment (stweil) [inline list.md]: I suggest _Niedersächsische Staats- und Universitätsbibliothek (SUB) Göttingen_.

**Comment by nilswindisch** [comment]
multitasking fail…

File: list.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ librarians.
 Library institutions, journals, and projects
 ---------------------------------------------
 
-SUB Göttingen
+Niedersächsische Staats- und Universitätsbibliothek Göttingen (SUB Göttingen, SUBUGOE)
```
Comment (stweil) [inline list.md]: I'd omit _SUBUGOE_ (or is that something used outside of GitHub)?

In addition, I suggest to squash all commits (combine them in a single commit).

Comment (nilswindisch) [inline list.md]: It's in use as twitter handle

Comment (hbunke) [inline list.md]: Could you please squash the commits, as @stweil suggests?

Comment (nilswindisch) [inline list.md]: GitHub's web UI doesn't support squashing commits or does it?

Comment (hbunke) [inline list.md]: Oh, you're right. GitHub has a (rather) new feature that allows that (https://github.com/blog/2141-squash-your-commits). Did'nt know that.

**Pull request merged**